src/blob-storage.js

```diff
@@ -61,19 +72,79 @@ export class BlobStorage {
    * @param {string} key - identifier of the blob
    * @returns {Promise<string>}
    */
-  async getSignedURI(key) {
+  async getSignedURI(key, expirationInSeconds = DEFAULT_DOWNLOAD_URI_TTL) {
     return getSignedUrl(
       this.#s3client,
       new GetObjectCommand({
         Bucket: this.#bucket,
         Key: key,
       }),
       {
-        expiresIn: 3600,
+        expiresIn: expirationInSeconds,
       },
     );
   }
 
+  async #createSignedPutURI(key, expiresIn, signableHeaders) {
```

Comment:
```suggestion
  async getSignedPutURI(key, expiresIn, signableHeaders) {
```

or maybe even:
```
async getSignedURI(key, method='GET', options)
```

Let's keep it simple for now. No need for the extra `generateUploadURIs` wrapper function.